Expensify/App
# Fix Issue #1339 Add pin icon to left hand navigation menu for chat rows that are pinned

cc @jboniface 

### Details
Added pin icon to pinned chats in the left side bar.

### Fixed Issues
Fixes #1339 

### Tests
1. Log into Expensify Cash
2. Pin a chat using the pin icon at the top right of the chat
3. Verify that small pin icon appears on the right hand side of the chat row in the left-hand nav.

### Tested On


### Screenshots
#### Web
<img width="1440" alt="Screenshot 2021-01-28 at 17 05 44" src="https://user-images.githubusercontent.com/3853003/106173230-9e5a5280-618b-11eb-93fe-da35c8bcfec4.png">

#### Mobile Web
<img width="582" alt="Screenshot 2021-01-28 at 17 06 30" src="https://user-images.githubusercontent.com/3853003/106173271-a6b28d80-618b-11eb-8951-d83c3ae17d01.png">

#### Desktop
<img width="1440" alt="Screenshot 2021-01-28 at 17 03 51" src="https://user-images.githubusercontent.com/3853003/106173298-aca86e80-618b-11eb-8422-0f7dbd760cef.png">

#### iOS
![Simulator Screen Shot - iPhone 11 - 2021-01-28 at 17 02 23](https://user-images.githubusercontent.com/3853003/106173333-b7630380-618b-11eb-93bf-1677eac652a6.png)

#### Android
![Screenshot_1611853006](https://user-images.githubusercontent.com/3853003/106173347-bc27b780-618b-11eb-8b32-becb503ce704.png)
